### DISCUSSION TITLE
Customize checkbox in drop down

### DISCUSSION THREAD

**Discussion by somya-wittybrains** [srivastavaanurag79/react-native-paper-select#21] [Q&A] [answered]
Hi,

I am trying to implement the react-native-paper-select for my app. I want to know if there is a way to customize the checkbox, for example instead of a checkbox we can use radio buttons or any icon.

**Comment by srivastavaanurag79** [comment]
No we don't have a support of it currently.

**Reply by somya-wittybrains** [reply]
Are you planning to implement this in the future?

**Reply by srivastavaanurag79** [reply]
Yes will let you know when it's done

**Reply by somya-wittybrains** [reply]
Thank you :)

**Comment by srivastavaanurag79** [comment, marked as answer]
Done
